src/components/BottomBar/BottomBar.jsx

```diff
@@ -17,6 +18,7 @@ const BottomBar = (props) => (
     <ExoplanetsPanel />
     <ActionsPanel />
     {props.showFlightController && <FlightControlPanel />}
+    <WWTPanel />
```

Comment:
Maybe call this `SkyBrowserPanel` instead? To better match the naming of the rest of the classes